configuration/items.md

```diff
@@ -242,40 +258,80 @@ FIRST_QUARTER=🌓 First Quarter
 ```
 
 ## Icons
-OpenHAB provides you a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
+OpenHAB provides a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
 However if you wish to use custom icons you need to place them inside the `conf/icons/classic/` folder.
 These icons will be used in all of the openHAB frontends.
-The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required).
-The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
 
-As an example, to use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>`.
+The images must be in `.png` or `.svg` format, consist of only lowercase letters, hyphens `-` or underscores `_` (if required).
```

Comment:
One more note we should mention is regarding dynamic icons and the [default iconset](http://docs.openhab.org/addons/iconsets/classic/readme.html). There is "error.svg" under them.

If I create an additional "error-no_fault.svg", it is not used.
If I create a copy ("error2.svg") and an "error2-no_fault.svg" file inside the `$CONF/icons/classic` folder, they are used just fine.

The message seems to be: predefined icons can not be "extended".